app/frontend/static/js/register/register.js

```diff
@@ -0,0 +1,70 @@
+const baseURL = "http://localhost:8000/"
+
+async function  logMeIn() {
+   let r;
+   let link = baseURL + "register"
+   var data = new FormData(regForm);
+   try {
+      let response = await fetch(link, {
+         method: "POST",
+         body: data,
+         headers: {
+         },
+         mode: 'no-cors'
+      })
+      console.log(response)
+      return response
+   } catch (error) {
+      console.log("error " + error)
+   }
+}
+
+const regForm = document.getElementById("regForm")
+const username = document.getElementById("username")
+const password = document.getElementById("password")
+const re_password = document.getElementById("floatingPassword_verif")
+const errorElement = document.getElementById('error')
+regForm.addEventListener("submit", regSubmit)
+
+
+async function regSubmit(event) { 
+   event.preventDefault()
+   // log.textContent = `Form Submitted! Timestamp: ${event.timeStamp}`;
+   console.log(username.value)
+   console.log(password.value)
+   console.log(re_password.value)
+  
+   let messages = [];
+
+   if (username.value === '' || username.value == null ) {
+      messages.push('Email is not valid');
+   }
+
+   if (password.value !== re_password.value) {
+      messages.push('Passwoards do not match')
+   }
+
+   if (messages.length > 0) {
+      errorElement.innerText = messages.join('\n')
+   } else {
+      let res = await logMeIn();
+
+      if (res.status === 401) {
+         console.log("Bad ")
```

Comment:
ar fi adecvat sa scoti console.log si mesajele de prin cand crezxio ca este gata ca nuu este ok sa intre lumea in consola sa vada astea